core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/TimeZoneTestRule.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.droidkaigi.confsched2023.testing
+
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+import java.util.TimeZone
+
+class TimeZoneTestRule : TestWatcher() {
+    override fun starting(description: Description) {
+        super.starting(description)
+        TimeZone.setDefault(TimeZone.getTimeZone("Asia/Tokyo"))
+    }
+}
```

Comment:
Global configuration may cause an problem so it should be reverted on finished anyway. And also, `super` don't need to be called in TestWatcher by the way.

```suggestion
class TimeZoneTestRule() : TestWatcher() {
    private lateinit var evacuatedTimeZone: TimeZone

    override fun starting(description: Description) {
        evacuatedTimeZone = TimeZone.getDefault()
        TimeZone.setDefault(TimeZone.getTimeZone("Asia/Tokyo"))
    }
    
    override fun finished(description: Description) {
        TimeZone.setDefault(evacuatedTimeZone)
    }
}
```

Reply:
Looks good! Thanks!